about.html

```diff
@@ -0,0 +1,117 @@
+            </ul>
+        </nav>
+    </header>
+<main>
+    <section class="descrption">
+    <h2>GLOBAL . STREAMING</h2>
+    <h2> BEST LIVE STREAMING WEBSITE -2022</h2>
+    <div>
+        <p>Global Streaming is a live-streaming website that allow users to record, 
+            broadcast video and stream it live simultaneously. Also it allows users to broadcast
+             live recordings of videos and audio. Others who have access to the Global Live video streaming,
+              have fueled influencer marketing, entertainment, business marketing, and the way people interact 
+              over the internet.
+             Global Streaming is the best website for sharing and streaming live event in 2022.</p>
+    </div>
+    <p>Please contact us via email for feedback or any other inquiry about Global Streaming</p>
+    <i>globalstreaming2022@gmail.com</i>
+    </section>
+    <section class="recent-events" id="event">
+     <h5>See recent streamed events</h5>
+     <hr>
+     <p> Take a look at three top past events which was streamed live here .....</p>
+     <div class="embeded-video">
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/c78H-nolqxI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/A_9m5INlV-M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+        </div>
+        <div class="video">
+          <iframe width="100%" height="270" src="https://www.youtube.com/embed/7WU__JQuB8g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+      </div>
+    </section>
+    <section class="other-categories">
+        <h4> Follow Us </h4>
+        <hr>
+        <ul>
+            <li> <span class="fa-brands fa-instagram"></span> Instagram</li> 
+            <li class="disable"><span class="fa-brands fa-twitter"></span> Twitter</li>
+            <li><span class="fa-brands fa-facebook"></span> Facebook</li>
+            <li class="disable"><span class="fa-brands fa-linkedin"></span> Linkedin</li>
+            <li><span class="fa-brands fa-telegram"></span> Telegram</li>
+        </ul> 
+      </section>
+</main>
+<footer>
+    <div>
+    <a href="#index.html"> <img src="./images/logo.png" alt="logo"> 
+        <p class="logo"> Global <span> Streaming</span> </p></a>
+    </div>
+    <div class="footer-copyright container">
+        <p>&copy; 2022 Global Streaming</p>
+        <p>Author : Progress C. Ezeamaka</p>
+      </div>
+</footer>
```

Comment:
- [ ] Please fix your footer, this is how it looks right now: 😄
  ![image](https://user-images.githubusercontent.com/20567503/198016655-2e567abf-7053-4508-9a8e-2f5c9946f4d4.png)

- [ ] Please know that the footer should have a dark background on desktop and white background on mobile, kindly fix this as well as it has a white background on desktop. 😉